README.md

```diff
@@ -6,4 +6,7 @@ Feel free to go wild here.
 
 ## The bikeshed.
 
-The [color of the bikeshed](https://bikeshed.com) is yet to be decided.
+<<<<<<< HEAD
```

Comment:
Oops that conflict marker should not be here, something went wrong with the conflict resolution.

Also, this is on a branch called "main", so your two PRs got mixed up here.